### DISCUSSION TITLE
Oxibooru Toolkit

### DISCUSSION THREAD

**Discussion by omni-custodian** [liamw1/oxibooru#2] [Ideas]
I have been working on a toolkit for szurubooru written in Rust based on the szurubooru api crate. If the api is the same as Szurubooru then I waas wondering if it would be fine to call mine oxibooru_toolkit to contrast it with the szurubooru toolkit that is written in python. My toolkit is really effecient at uploading posts and editting tags, but I'm really excited to see how fast it can really be on oxibooru. What are your thoughts?

**Comment by liamw1** [comment]
Sounds good to me! The API is almost exactly the same as szurubooru. The only exception is [Listing Snapshots](https://github.com/rr-/szurubooru/blob/master/doc/API.md#listing-snapshots), which doesn't exist in oxibooru.
